src/vconsole/vconsole-setup.c

```diff
@@ -330,3 +330,5 @@ int main(int argc, char **argv) {
 
         return font_ok && keyboard_ok ? EXIT_SUCCESS : EXIT_FAILURE;
 }
+
+/* vim: set ts=8 sts=8 sw=8 et : */
```

Comment:
Oh, it grepped for "ts=", but .vimrc has the name expanded to tabstop, so I thought we don't have a global file with vim settings. Yeah, those settings duplicate what is already in .vimrc, let's drop this.